spec/unit/lcm/lcm2_spec.rb

```diff
@@ -0,0 +1,36 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'gooddata/lcm/lcm2'
+
+describe 'GoodData::LCM2' do
+  describe '#skip_actions' do
+    let(:client) { double(:client) }
+    let(:domain) { double(:domain) }
+    let(:logger) { double(:logger) }
+    let(:params) do
+      params = {
+        skip_actions: %w[CollectSegments SynchronizeUsers],
```

Comment:
Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `(` and `)`.